qcoptions.py

```diff
@@ -21,4 +21,9 @@
                   help="[OPTIONAL] Run a shortened test after covers have been applied")
 parser.add_option("--ztrim", type="float", dest="ztrim", default=0.0,
                   help="Specify the p value of the trim", metavar="ztrim")
-
+parser.add_option("--scanmin", type="int", dest="scanmin",
```

Comment:
I am not sure I am a fan of scanmin and scanmax being configurable, we should force the user to  scan over the whole range, and have a debug mode which does a shorter scan